library/wsgiref.po

```diff
@@ -143,18 +172,27 @@ msgid ""
 "*`` variables.  It only supplies default values, and does not replace any "
 "existing settings for these variables."
 msgstr ""
+"Cette fonction ajoute des paramètres requis pour WSGI, en particulier "
+"``HTTP_HOST``, ``SERVER_NAME``, ``SERVER_PORT``, ``REQUEST_METHOD``, "
+"``SCRIPT_NAME, ``PATH_INFO`` et les autres variables wsgi définies dans la :"
+"pep:`3333`. Elle ne fournit que des valeurs par défaut sans toucher aux "
+"valeurs déjà définies de ces variables."
 
 #: ../Doc/library/wsgiref.rst:113
 msgid ""
 "This routine is intended to make it easier for unit tests of WSGI servers "
 "and applications to set up dummy environments.  It should NOT be used by "
 "actual WSGI servers or applications, since the data is fake!"
 msgstr ""
+"Cette fonction a uniquement pour but de faciliter les tests unitaires "
+"des serveurs et des applications WSGI dans des environnements factices. Elle ne "
+"devrait pas être utilisée dans une application ou un serveur WSGI, étant donné "
+"que les données sont factices !"
```

Comment:
```suggestion
"que les données sont factices !"
```

/tmp/padpo_devj0_41/library/wsgiref.po: 181 [     NBSP     ] There should be a non-breakable space before "!": between ### que les données sont factices ### and ###!###